evap/student/templates/student_vote.html

```diff
@@ -141,15 +141,19 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
                     url: form.attr('action'),
                     data: form.serialize(),
                     success: function (data, s, req) {
-                        sisyphus.manuallyReleaseData(); // delete the data from local storage
-                        // Manually perform the redirect: first replace content
-                        document.open();
-                        document.write(data);
-                        document.close();
-                        // then set the URL
-                        window.history.pushState('', '', xhr.responseURL);
-                        // scroll to the top of the page
-                        $('html,body').scrollTop(0);
+                        if(data === "{{ success_magic_string }}") {
+                            sisyphus.manuallyReleaseData(); // delete the data from local storage
+                            window.location.replace("{{ success_redirect_url }}");
+                        } else {
+                            // Manually perform the redirect: first replace content
```

Comment:
maybe "manually perform the page load: first replace content", that would be less wrong